optimum/neuron/models/training/modeling_utils.py

```diff
@@ -1258,7 +1281,7 @@ def from_pretrained(
         config.name_or_path = pretrained_model_name_or_path
 
         # Instantiate model.
-        init_contexts = [no_init_weights(_enable=_fast_init)]
+        init_contexts = [no_init_weights()]
```

Comment:
Why?

Reply:
Ok, we might need to check if this is assumed or enabled in the `_load_pretrained_weights` method.